sdk/azidentity/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### Features Added
 * Service principal credentials can request CAE tokens
 
+- Added Azure Developer CLI credential ([#20610](https://github.com/Azure/azure-sdk-for-go/issues/20610))
```

Comment:
```suggestion
* Added Azure Developer CLI credential ([#20610](https://github.com/Azure/azure-sdk-for-go/issues/20610))
```

Reply:
@chlowell We should add this credential type to the DAC auth flow, immediately after the Azure CLI credential. Do you plan to make that change in a separate PR?

Reply:
I'll do it in this one